components/footer/footer.scss

```diff
@@ -22,45 +22,58 @@
       justify-content: space-between;
     }
 
-    p {
-      margin-top: 0;
-      margin-bottom: $base;
-    }
-
     &-col {
       display: flex;
       flex-direction: column;
-      color: $secondary;
       line-height: 20px;
 
       .links {
-        margin-bottom: $base;
+        display: flex;
+
+        @include xsm {
+          flex-direction: column;
```

Comment:
i think we should also add `align-items: center` here so that the links aren't 100% width on mobile